applications/ExternalSolversApplication/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 message("**** configuring KratosExternalSolversApplication ****")
 
 if(${INCLUDE_PASTIX} MATCHES ON)
-	set(${EXCLUDE_ITSOL} ON)
```

Comment:
i am personally in favor of completely removing ITSOL...(your patch is fine though)